config/config.js

```diff
@@ -9,7 +11,13 @@ module.exports = function (options) {
       port: process.env.POSTGRES_PORT || 5432,
       username: process.env.POSTGRES_USERNAME,
       password: process.env.POSTGRES_PASSWORD,
-      nolimit: true
+      nolimit: true,
+      fromColumnName: (attr) => {
+        return camelCase(attr);
```

Comment:
i feel you know this isnt what i meant 😜

Reply:
ah, fair, can still reduce it
Will do that tomorrow